src/go/cmd/update-collection-v3/main.go

```diff
@@ -43,34 +42,31 @@ func migrateYamlFile(yamlV2FilePath string, yamlV3FilePath string) error {
 		return fmt.Errorf("error migrating values %v", err)
 	}
 
-	err = ioutil.WriteFile(yamlV3FilePath, []byte(yamlV3), fs.ModeType)
+	err = ioutil.WriteFile(yamlV3FilePath, []byte(yamlV3), 0666)
```

Comment:
```suggestion
	err = ioutil.WriteFile(yamlV3FilePath, []byte(yamlV3), os.ModePerm)
```

it is for 0777 but looks nicer :sweat_smile:

Reply:
I don't agree with this one. I don't think this is the idiomatic use for `os.ModePerm`.